### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by elmar8287**
- Set up a new project with webpack.
- Created HTML, JS, and CSS files according to project requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+Tasks to do:
+
+- Project 1: List structure.
+- Project 2: Interactive list.
+- Project 3: Add and remove items.
+
+![Screen Shot]()
```
Comment (MahmoudBakr23) [inline README.md]: - [x] Kindly fix this broken image for completion of the professional README. 😊

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  background-color: rgb(235, 235, 235);
+}
+
+#ppp {
+  border-bottom: solid rgb(241, 241, 241);
+  margin: 4px;
+  padding: 4px;
+}
+
+/* .js-todo-list {
```
Comment (MahmoudBakr23) [inline src/style.css]: - [x] **OPTIONAL**
For cleaner code and better optimization, please remember to remove any commented / unused code from your files.
_Kindly note that this point will be learned and be asked about in the upcoming projects. 😊_

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @elmar8287 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 
- Well done using webpack configurations to build the files
- Descriptive pr summary and headline 
- Live demo link 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (elmar8287) [reply README.md]: Fixed

File: src/style.css (hunk 2 above)
Comment (elmar8287) [reply src/style.css]: Fixed

**Comment by elmar8287** [comment]
Dear @MahmoudBakr23,

Thank you for your review. I have fixed all your recomentations.

BR

Elmar

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @elmar8287. 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — COMMENTED** [review]
### **Hello @elmar8287** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has Already been approved. Please submit the right PR link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**